src/Ticket.java

```diff
@@ -33,6 +40,7 @@ public Ticket(String id,
         this.isPromo = isPromo;
         this.sector = sector;
         this.maxBackpackWeightInKg = maxBackpackWeightInKg;
+        NullableWarningChecker.check(this);
```

Comment:
validate before creating (in this case just move it to 36 line)

Reply:
done